tests/core/meson.build

```diff
@@ -68,9 +68,26 @@ if host_arch == 'arm64'
   endif
 endif
 
+gum_tests_core_link = []
```

Comment:
Let's rename this to `core_link_with` (or something like that) and move this up right after `core_sources`, with a blank line before it.